src/jetstream/main.go

```diff
@@ -773,6 +784,10 @@ func (p *portalProxy) registerRoutes(e *echo.Echo, addSetupMiddleware *setupMidd
 	pp.GET("/v1/auth/sso_login", p.initSSOlogin)
 	pp.GET("/v1/auth/sso_logout", p.ssoLogoutOfUAA)
 
+	// Local User login/logout
+	pp.POST("/v1/auth/local_login", p.localLogin)
+	pp.POST("/v1/auth/local_logout", p.logout)
```

Comment:
Do you think the local routes and remote routes should check if the corresponding auth type matches and response with 404 ?

Reply:
Yes. Will add.